packages/server/src/orders/database/OrderSchema.ts

```diff
@@ -0,0 +1,75 @@
+import { Prop, raw, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { ObjectId } from 'mongodb';
+import { Document } from 'mongoose';
+
+import { Address } from '../../addresses/database/AddressSchema';
+import { User } from '../../users/database/UserSchema';
+
+export type OrderDocument = Order & Document;
+
+const ORDER_CONSTANTS = Object.freeze({
+  COMMENT: Object.freeze({
+    MAX_LENGTH: 300,
+  }),
+  PRICE: Object.freeze({
+    MIN: 0,
+  }),
+  COUNT: Object.freeze({
+    MIN: 1,
+  }),
+});
+
+class SubOrder {
+  @Prop({ required: true })
+  deliveryDate: Date;
+
+  @Prop({ required: true })
+  hourStart: number;
+
+  @Prop({ required: true })
+  hourEnd: number;
+
+  @Prop(
+    raw({
+      dishId: {
+        type: [{ type: ObjectId, ref: 'Dish' }],
+        required: true,
+      },
+      price: { type: Number, min: ORDER_CONSTANTS.PRICE.MIN },
+      count: { type: Number, default: ORDER_CONSTANTS.PRICE.MIN, min: ORDER_CONSTANTS.PRICE.MIN },
+      excludedIngredients: { type: [String] },
+    }),
+  )
+  dishes: object[];
```

Comment:
DishSchema już jest na main, to możesz zaimportować Scheme i dać:
`dishes: Dishes[]` tak jak tam niżej zrobiłaś :D